src/App.jsx

```diff
@@ -14,70 +15,84 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {(selectedGood === '')
+        ? (
+          <h1
+            className="title is-flex is-align-items-center"
+          >
+            No goods selected
+          </h1>
+        )
+        : (
+          <h1 className="title is-flex is-align-items-center">
```

Comment:
In both cases, you have the same h1 with the same classname. Consider moving ternary operator inside the tag